includes/nf/ziti.h

```diff
@@ -179,6 +179,38 @@ typedef nf_conn_cb nf_listen_cb;
  */
 typedef void (*nf_write_cb)(nf_connection conn, ssize_t status, void *write_ctx);
 
+/**
+ * @brief Callback called after NF_enroll() is complete.
+ *
+ * This callback is invoked on the conclusion of the NF_enroll() function. The result of the
+ * NF_enroll() function may be an error condition so it is important to verify the provided
+ * status code in this callback.
+ *
+ * This callback also returns a Ziti identity json salvo if the enrollment was successful. 
```

Comment:
This callback _returns_... should be _receives_?

Reply:
Yup. fixed.